cloudformation/elk-stack.template

```diff
@@ -263,6 +263,11 @@
                         "Protocol": "TCP",
                         "LoadBalancerPort": "6379",
                         "InstancePort": "6379"
+                    },
+                    {
+                        "Protocol": "TCP",
+                        "LoadBalancerPort": "80",
+                        "InstancePort": "80"
```

Comment:
May we create a separate security group for this rule as `Allow logstash messages to internal ELB` does not really match any more with the new rule